exercises/concept/ellens-alien-game/.docs/hints.md

```diff
@@ -0,0 +1,22 @@
+# Hints
+
+## 1. Create the Alien Class
+
+- The `constructor` does not have a return type and has the same name as the `class`.
+- Usually, the names of `classes` start with a capital letter.
+
+## 2. The `hit` Method
+
+- Remember the invariant: health points can't be negative.
+
+## 3. The `is_alive` Method
+
+- life points at `0` is the only 'dead' condition.
+
+## 4. The `teleport` Method
+
+- Member variables can be updated from any method from within the class.
+
+## 5. The `collision_detection` Method
+
+- All `public` members variables can be called and changed from other classes.
```

Comment:
member variables can be "called"?  And what about non-class contexts?
Instead, how about "All \`public\` member variables can be accessed from from outside the class."